powerdns/models/powerdns.py

```diff
@@ -390,6 +390,53 @@ class Meta:
         verbose_name = _("record")
         verbose_name_plural = _("records")
 
+    def request_factory(operation):
```

Comment:
and since this is returning links in admin context, why this is attached directly to model?